docs/UserGuide.md

```diff
@@ -3,64 +3,65 @@ layout: page
 title: User Guide
 ---
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+LE TRACKER is a gamified tracking application that allows fast typist to easily log their lecture progress, search for lecture by mod code/ keywords/ topics for a stress-free learning environment. Unlike todo list applications, LE TRACKER is tailored to the needs of students; it provides additional information specific to lecture media such as watch progress and topics.
 
-* Table of Contents
-{:toc}
+- Table of Contents
+  - [Quick Start](#quick-start)
 
---------------------------------------------------------------------------------------------------------------------
+---
 
-## Quick start
+## Quick Start
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
-
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
-
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar addressbook.jar` command to run the application.<br>
-   A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
-
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
-   Some example commands you can try:
-
-   * `list` : Lists all contacts.
-
-   * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
-
-   * `delete 3` : Deletes the 3rd contact shown in the current list.
-
-   * `clear` : Deletes all contacts.
-
-   * `exit` : Exits the app.
+2. Type the command in the command box.
+
+3. Press Enter to execute it. e.g. typing help and pressing Enter will open the help window.
+
+- List
+  - `list /module`: Lists the names of all the recorded modules.
+  - `list /module /{module_code}`: Lists all the videos recorded in that module code
+- Add
+  - `add-module {module_code} [/name {module_name}]`: Adds a module to Le Tracker
+  - `add-lecture /module {module_code}`: Adds a lecture to a module
+  - `add-video /module {module_name} /lecture {lecture_index} /video {video_name}`: Adds a video to the module code
+- Mark/Unmark
+  - `mark /module {module_code} /lecture {lecture_index} /video {video_name}`: Marks a video as watched
+  - `unmark /module {module_code} /lecture {lecture_index} /video {video_name}`: Unmarks a video as unwatched
```

Comment:
Perhaps change "Unmarks a video as unwatched" to "Marks a video as not watched"?